lib/_http_outgoing.js

```diff
@@ -705,6 +706,11 @@ function write_(msg, chunk, encoding, callback, fromEnd) {
     return true;
   }
 
+  if (!fromEnd && typeof chunk !== 'string' && !isUint8Array(chunk)) {
+    throw new ERR_INVALID_ARG_TYPE('chunk',
+                                   ['string', 'Buffer', 'Uint8Array'], chunk);
+  }
```

Comment:
This is duplicate https://github.com/nodejs/node/pull/33155/files#diff-286202fdbdd74ede6f5f5334b6176b5cL678